.github/workflows/main.yml

```diff
@@ -29,12 +29,44 @@ jobs:
           path: "**/TEST-*.xml"
           testSrcPath: "**/test/"
 
+  android-tests:
```

Comment:
#L12 should be renamed from `build` to `tests` or `java-tests`

Reply:
done, renamed to java-tests